nano/node/node.cpp

```diff
@@ -1187,22 +1187,9 @@ void nano::node::process_confirmed (nano::election_status const & status_a, uint
 	auto transaction (store.tx_begin_read ());
 	if (store.block_get (transaction, hash, &sideband) != nullptr)
 	{
-		confirmation_height_processor.add (hash);
-
-		receive_confirmed (transaction, block_a, hash);
-		nano::account account (0);
-		nano::uint128_t amount (0);
-		bool is_state_send (false);
-		nano::account pending_account (0);
-		process_confirmed_data (transaction, block_a, hash, sideband, account, amount, is_state_send, pending_account);
-		observers.blocks.notify (status_a, account, amount, is_state_send);
-		if (amount > 0)
+		if (status_a.type == nano::election_status_type::active_confirmed_quorum)
```

Comment:
Probably this can be before read transaction. Also if `type != nano::election_status_type::active_confirmed_quorum` then `else if (iteration < 20)` part seems not required

Reply:
sounds good :)